src/main/java/com/minecraftabnormals/upgrade_aquatic/common/entities/jellyfish/ColoredSizableJellyfishEntity.java

```diff
@@ -55,9 +62,9 @@ public void readAdditional(CompoundNBT compound) {
 	public ILivingEntityData onInitialSpawn(IServerWorld worldIn, DifficultyInstance difficultyIn, SpawnReason reason, ILivingEntityData spawnDataIn, CompoundNBT dataTag) {
 		spawnDataIn = super.onInitialSpawn(worldIn, difficultyIn, reason, spawnDataIn, dataTag);
 		boolean updateSize = false;
-		float size = this.getRNG().nextFloat() < 0.85F ? 1.0F : this.getRNG().nextBoolean() ? 0.5F : 0.65F;
-		int color = this.getRNG().nextInt(3);
 
+		int color = this.getRNG().nextInt(3);
+		float size = this.getSizeMap().randomSize(this.getRNG());
```

Comment:
`this.getRNG()` is called twice in these two lines. Move it to a local variable.

Reply:
Fixed in latest commit.